geometry_msgs/PoseStamped pose 
duration loiter
uint32 id